patches/setenv.patch

```diff
@@ -66,15 +66,15 @@ index 00000000..4c28cdab
 +# so as to pull a fresh copy.  If this is the first time building
 +# depot tools, these file will not be present and thus
 +# no need to attempt to remove them.
-+# NOTE: assuming v8 will be built in $HOME/zopen/dev/v8base
++# NOTE: assuming v8 will be built in $HOME/zopen/dev/v8port/v8base
 +if [ ${RESUME} -eq 0 ]; then
 +    echo "Attempting to delete any leftover .gclient* entries"
-+    if [ -d "${HOME}/zopen/dev/v8base" ]; then
-+        if [ -f "${HOME}/zopen/dev/v8base/.gclient_entries" ]; then
-+            rm ${HOME}/zopen/dev/v8base/.gclient_entries
++    if [ -d "${HOME}/zopen/dev/v8port/v8base" ]; then
++        if [ -f "${HOME}/zopen/dev/v8port/v8base/.gclient_entries" ]; then
++            rm ${HOME}/zopen/dev/v8port/v8base/.gclient_entries
 +        fi
-+        if [ -f "${HOME}/zopen/dev/v8base/.gclient_entries" ]; then
-+            rm ${HOME}/zopen/dev/v8base/.gclient
++        if [ -f "${HOME}/zopen/dev/v8port/v8base/.gclient" ]; then
++            rm ${HOME}/zopen/dev/v8port/v8base/.gclient
 +        fi
 +    fi
 +fi
```

Comment:
Is a `patches/setenv.patch` to patch a file `setenv.sh` in the same repo required? I suggest removing it and update `setenv.sh` directly.

Also suggest setting a variable `V8BASE="${HOME}/zopen/dev/v8port/v8base"` in `setenv.sh` and using that variable; e.g. `if [ -f "$V8BASE/.gclient_entries" ]; then`.

Reply:
Igor wants to move this stuff to v8.  fwiw, i have two setenv's.  one is a script that can be used during development for quick and easy access.  The other is the patch which gets added to the install directory.  Once its installed, it can be used from that place.

However, with that said, its removed from this project and it wll be moved to the v8 project.

Lastly, I am trying to use the ZOPEN variables.